src/inet/TCPEndPoint.cpp

```diff
@@ -2368,15 +2372,15 @@ void TCPEndPoint::HandlePendingIO()
     // ready to be received on the socket, process the incoming connection.
     if (State == kState_Listening)
     {
-        if (OnConnectionReceived != nullptr && mPendingIO.IsReadable())
+        if (OnConnectionReceived != nullptr && mPendingIO.Has(System::SocketEventFlag::kRead))
```

Comment:
Any reason to not make `IsReadable()` as a convenience inline for `Has(System::SocketEventFlag::kRead)`?

Reply:
The pattern for most other flag sets in the code is not to have them, so I didn't re-add them when moving `SocketEvents` to use the type-safe `BitFlags` when I had to move it from inet to system. (A bit sad that enum class can't have methods, not to mention a bitwise mixin.)